### PR TITLE
Issue 3636

### PR DESCRIPTION
**Pull request by BenFradet**

### CODE REVIEW

File: 3-enrich/scala-common-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/enrichments/registry/PiiPseudonymizerEnrichment.scala [hunk 1]
```diff
@@ -357,7 +357,11 @@ final case class PiiJson(strategy: PiiStrategy,
       jsonPath,
       ScrambleMapFunction(strategy)
     )
-    JsonMethods.fromJsonNode(documentContext.json[JsonNode]())
+    // make sure it is a structure preserving method, see #3636
+    val transformedJValue            = JsonMethods.fromJsonNode(documentContext.json[JsonNode]())
+    val Diff(_, erroneouslyAdded, _) = jValue diff transformedJValue
+    val Diff(_, wihtoutCruft, _)     = erroneouslyAdded diff transformedJValue
```
Comment (knservis) [inline 3-enrich/scala-common-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/enrichments/registry/PiiPseudonymizerEnrichment.scala]: Typo

**Comment by knservis** [comment]
@BenFradet Let's move this to the issue-3636 branch

**Comment by BenFradet** [comment]
it's in the issue-3636 branch? :thinking:

**Comment by knservis** [comment]
Sorry @BenFradet.. I meant in the release/gdpr-2 branch (after fixing the commit messages)

**Comment by BenFradet** [comment]
>Sorry @BenFradet.. I meant in the release/gdpr-2 branch (after fixing the commit messages). We can change the PR to that branch instead of master

ah ok, yes feel free to do it :+1:

File: 3-enrich/spark-enrich/build.sbt [hunk 2]
```diff
@@ -16,7 +16,7 @@ lazy val root = project
   .in(file("."))
   .settings(
     name := "snowplow-spark-enrich",
-    version := "1.13.0",
+    version := "1.14.0",
```
Comment (chuwy) [inline 3-enrich/spark-enrich/build.sbt]: Shouldn't it be bugfix 1.13.1 release?

**Pull request merged**